Adding Solid World Coordinator role (#284)
solid/process: README.md

Administrators belong to the [Administrators Team](https://github.com/orgs/solid/teams/administrators) in the [Solid GitHub Organization](https://github.com/solid) and have [_Admin Permissions_](https://help.github.com/en/articles/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization) on all repositories therein. Administrators have [_Owner Permissions_](https://help.github.com/en/articles/permission-levels-for-an-organization#permission-levels-for-an-organization) for the [Solid GitHub Organization](https://github.com/solid).

> The Solid World Coordination Administrator is granted privileged access to only those tools, systems, and services required to coordinate and promote the monthly Solid World webinar, including Vimeo, Twitter, Eventbrite and Typeform.

Reviewer: It might be worthwhile to add some language in here that doesn't specifically lock in the tools, so that we don't have to change the process if we decide to switch from (for example) Eventbrite or Typeform. It might be easier to remove the specifics altogether, and then it can be left up to the team to determine what's necessary. Otherwise looks good to me.

> The Solid World Coordination Administrator is granted privileged access to only those tools, systems, and services required to coordinate and promote the monthly Solid World webinar, including Vimeo, Twitter, Eventbrite and Typeform.

Reviewer 2: ```suggestion
The Solid World Coordination Administrator is granted privileged access to only those tools, systems, and services required to coordinate and promote the monthly Solid World webinar, such as Vimeo, Twitter, Eventbrite, and/or Typeform.
```

Administrators belong to the [Administrators Team](https://github.com/orgs/solid/teams/administrators) in the [Solid GitHub Organization](https://github.com/solid) and have [_Admin Permissions_](https://help.github.com/en/articles/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization) on all repositories therein. Administrators have [_Owner Permissions_](https://help.github.com/en/articles/permission-levels-for-an-organization#permission-levels-for-an-organization) for the [Solid GitHub Organization](https://github.com/solid).

> The Solid World Coordination Administrator is granted privileged access to only those tools, systems, and services required to coordinate and promote the monthly Solid World webinar, such as Vimeo, Twitter, Eventbrite, and/or Typeform.

Reviewer 2: ```suggestion
The Solid World Coordination Administrator is granted privileged access to such tools, systems, and services as are required to coordinate and promote the monthly Solid World webinar, such as Vimeo, Twitter, Eventbrite, Typeform, and/or others.
```
